ui/imports/shared/panels/HoldingTypes.qml

```diff
@@ -0,0 +1,7 @@
+import QtQml 2.14
+
+QtObject {
```

Comment:
maybe this can go into Constants.qml file instead? other enums are defined there, maybe we can improve and create a constants file related to send  as an improvement later

Reply:
can do

Reply:
done